src/collections/_documentation/accounts/quotas/index.md

```diff
@@ -3,20 +3,23 @@ title: 'Quotas & Events'
 sidebar_order: 0
 ---
 
-Events and quotas are interconnected in Sentry. At the most basic, when you [subscribe to Sentry](https://sentry.io/pricing/), you pay for the number of events to be tracked. This number of events is your quota. When an event is accepted by Sentry, it counts toward your quota.
+Events and quotas are interconnected in Sentry. At the most basic, when you [subscribe to Sentry](https://sentry.io/pricing/), you pay for the number of events - errors, attachments, and transactions - to be tracked. Each of these type of events has a quota. When an event is accepted by Sentry, it counts toward your quota for that type of event.
 
 Sentry’s flexibility means you can exercise fine-grained control over which events count toward your quota.
 
 ## Key Terms
 
 Let’s clarify a few terms to start:
+
 -   Event - an event is one instance of you sending Sentry data. Generally, this data is an error. Every event has a set of characteristics, called its fingerprint.
--   Issue - an issue is a grouping of similar events, who all share the same fingerprint. For example, Sentry groups events together when they are triggered by the same part of your code. For more information, see [Grouping & Fingerprinting](https://docs.sentry.io/data-management/event-grouping/).
--   Quota - your quota is the monthly number of events you pay Sentry to track.
+-   Issue - an issue is a grouping of similar events, which all share the same fingerprint. For example, Sentry groups events together when they are triggered by the same part of your code. For more information, see [Grouping & Fingerprinting](https://docs.sentry.io/data-management/event-grouping/).
```

Comment:
```suggestion
-   Issue - an issue is a grouping of similar events, which all share the same fingerprint. For example, Sentry groups events together the same part of your code triggers them. For more information, see [Grouping & Fingerprinting](https://docs.sentry.io/data-management/event-grouping/).
```

Reply:
I'm not sure that edit clarifies.

Reply:
Ah, you're right. I think I meant, "For example, Sentry groups events together when the same part of your code triggers them." But, I think the original version also works. Was trying to work it into active voice.

Reply:
In some cases, we gotta go with the passive voice :)